hr_expense_sequence/README.rst

```diff
@@ -0,0 +1,35 @@
+Sequence for HR expense
+=======================
+
+This module gives a unique number for each HR expense declaration.
+
+Configuration
+=============
+
+You can change the default sequence (EX0000...) by the one of your choice
+going to Settings > Technical > Sequences & Identifiers > Sequences, and
+editing _HR expense_ record.
+
+You will only have access to that section if your section has _Technical
+features_ permission check marked.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+
```

Comment:
It seems that this PR mixes one of my modules, so he has to remove it, not to attribute it.